app.js

```diff
@@ -0,0 +1,17 @@
+const  { getTopics } = require("./controllers");
+const express = require("express");
+const app = express();
+
+
+app.use(express.json());
+app.get("/api/topics", getTopics);
+
+app.use((err, req, res, next) =>{
+    res.status(err.status).send({ msg: err.msg});
```

Comment:
You should make a check here, something like `if (err.status && err.msg)` and only send it if it passes the condition. Then make sure `next(err)` is inside an else block.

Reply:
I have done